src/components/SearchMap.js

```diff
@@ -56,20 +65,94 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate})
         updateRegion(newRegion);
     };
 
+    const renderIcons = ({features, facilities}) => {
+        const featureIcons = _.map(features, feature => {
+            return (
+                <Icon
+                    key={feature}
+                    reverse
+                    size={15}
+                    name={featureIconDetails[feature].name}
+                    type={featureIconDetails[feature].type}
+                />
+            )
+        });
+
+        const facilityIcons = _.map(facilities, facility => {
+            return (
+                <Icon
+                    key={facility}
+                    reverse
+                    size={15}
+                    name={facilityIconDetails[facility].name}
+                    type={facilityIconDetails[facility].type}
+                />
+            );
+        });
```

Comment:
This `renderIcons` is VERY similar to the logic used in the `CampsiteListItem` component. We should probably refactor this into a service or something, but I'll have to think a little more about how to make it properly dynamic.